src/Styling/SubMenu.css

```diff
@@ -29,12 +25,61 @@
     align-items: center;
     text-decoration: none;
     color: #f5f5f5;
-    font-size: 18px;
 }
 /* TODO; find the difference between background color and background and when to use it */
 .Dropdown_Link:hover {
-    background: #DE5B31;
+    background-color: #DE5B31;
     cursor: pointer;
 }
 
+.DropDown_Indicator {
+  margin: 6px 140px 0px 0px;
+}
+
+/* Media Query */
+/* screen size 768 or lower */
+@media only screen and (max-width: 768px) {
```

Comment:
check indentation within this media query. inconsistent indentation to other queryies